### ISSUE TITLE
Feature Request - Option to Disable Links or Display full URL in message body

### ISSUE THREAD

**Issue by Subterrane** [leggett/simplify#302]
My company likes to send out test phishing emails and they happened to send one about AWS while I was having a Slack conversation about AWS so my stupid pea-brain went and clicked on the link and I failed the test (they're nice about it, but it's embarrassing).

Every once in a while, I look all over for a simple email client that would just display the plaintext mime part, but I think it might be easier let HTML display and:

a) Select all the links in the mail DOM and preventDefault 
   or
b) Grab the link URL and expose it in the message body right next to the link. Example:
 __Click Me!__ [http://evilhacker.com/sendmeyourbitcoins]

Basically add a "don't shoot your own foot" option. It might be simpler to figure it out and write my own extension but since you're in there already... 😄

**Comment by leggett** [comment]
Sorry you got phished. Glad it was only a test and not a real phishing attack. 

I'd certainly love to help prevent phishing attacks if I think I reliably can. But I don't see a viable feature here. Maybe I'm just not thinking of the right approach yet...

- Inserting the links into the body like that is going to really mess with the flow of reading the email. Especially if it is a long link with a lot of url params. 
- I played with showing the link on hover but the browser already does that in the bottom left corner. 
- I could try to scan for links that show a URL that doesn't match the domain of the URL the link goes to, but it might still be a legit link going through some kind of link click tracking software platform that redirects to the used URL. So you'd learn to ignore this too and it wouldn't help when the link text wasn't a url.

**Comment by Subterrane** [comment]
> Inserting the links into the body like that is going to really mess with the flow of reading the email. Especially if it is a long link with a lot of url params.

Indeed, but I kind of want that.

> I played with showing the link on hover but the browser already does that in the bottom left corner.

Yeah, you're right. I was using the Mac mail client when I got click-happy. I should stick with the browser. 

> I could try...

Thanks for taking the time to consider it. Offhand, I can't really think of anything better than the ugly options. Even the domain matching wouldn't work since people register domains that look legit at first glance. 

I'm kind of surprised Gmail isn't better about this since they are scanning everything for spam anyway. I'll keep pointing fingers at everyone else until I feel better about being dumb.

**Comment by leggett** [comment]
Gmail definitely scans for phishing and let's you report messages as phishing. 

But I think phishing tests like this inside companies are rarely caught.
